### PR TITLE
Change terraform data according to "terraform-provider-kubevirt" changes

### PR DESCRIPTION
**Pull request by nirarg**

### CODE REVIEW

File: data/data/kubevirt/bootstrap/main.tf [hunk 1]
```diff
@@ -35,18 +35,100 @@ resource "kubernetes_secret" "bootstrap_ignition" {
...
+      }
+    }
+    template {
+      metadata {
+        labels = {
+          "kubevirt.io/vm" = "${var.cluster_id}-bootstrap"
+        }
+      }
+      spec {
+        volumes {
```
Comment (ravidbro) [inline data/data/kubevirt/bootstrap/main.tf]: Shouldn't 'volumes' needs to be from array type and then 'volume' objects in that list?

File: data/data/kubevirt/bootstrap/main.tf [hunk 2]
```diff
@@ -35,18 +35,100 @@ resource "kubernetes_secret" "bootstrap_ignition" {
...
+        }
+        domain {
+          resources {
+            requests = {
+              memory = var.memory
+              cpu    = var.cpu
+            }
+          }
+          devices {
+            disks {
```
Comment (ravidbro) [inline data/data/kubevirt/bootstrap/main.tf]: same question as 'volumes'

File: data/data/kubevirt/bootstrap/main.tf [hunk 3]
```diff
@@ -35,18 +35,100 @@ resource "kubernetes_secret" "bootstrap_ignition" {
...
+            }
+            disks {
+              name = "${var.cluster_id}-bootstrap-cloudinitdisk"
+              disk_device {
+                disk {
+                  bus = "virtio"
+                }
+              }
+            }
+            interfaces {
```
Comment (ravidbro) [inline data/data/kubevirt/bootstrap/main.tf]: same, interface or interfaces?

File: data/data/kubevirt/bootstrap/main.tf (hunk 1 above)
Comment (nirarg) [reply data/data/kubevirt/bootstrap/main.tf]: Here I'm using nested block syntax (attribute as block):
https://www.terraform.io/docs/configuration/attr-as-blocks.html

Can be more than one block, it used as array

Comment (ravidbro) [reply data/data/kubevirt/bootstrap/main.tf]: fine, so each block name should be singular and not plural

Comment (nirarg) [reply data/data/kubevirt/bootstrap/main.tf]: Changed - https://github.com/nirarg/installer/pull/4/commits/1911a77f0238b3dcbf2e3f88b02bf988eb28a314

File: data/data/kubevirt/bootstrap/main.tf (hunk 2 above)
Comment (nirarg) [reply data/data/kubevirt/bootstrap/main.tf]: Changed - 1911a77

File: data/data/kubevirt/bootstrap/main.tf (hunk 3 above)
Comment (nirarg) [reply data/data/kubevirt/bootstrap/main.tf]: Changed - 1911a77

**Pull request merged**